services/discord/discord.service.js

```diff
@@ -37,16 +45,22 @@ module.exports = class Discord extends BaseJsonService {
   static get route() {
     return {
       base: 'discord',
-      pattern: ':serverId',
+      pattern: ':serverId/:inviteId?',
     }
   }
 
   static get examples() {
     return [
       {
-        title: 'Discord',
+        title: 'Discord Server',
```

Comment:
The example titles don't really convey what the badges will be displaying. Would the following be more informative?
* Discord Online Members
* Discord Online & Total Members

Feel free to suggest something else as well! :)

Reply:
I like the `Discord Online & Total Members`, does it make sense to merge the examples into 1? Since the invite id is optional, and when it's not added it will show just like the old one

Reply:
As you like, I think it's also fine to keep the two examples!